include/pinocchio/parsers/urdf.hpp

```diff
@@ -42,6 +43,7 @@ namespace pinocchio
       const typename ModelTpl<Scalar, Options, JointCollectionTpl>::JointModel & rootJoint,
       const std::string & rootJointName,
       ModelTpl<Scalar, Options, JointCollectionTpl> & model,
+      const bool mimic = false,
```

Comment:
Saddly, this also break the API. Call that set verbose to true will set mimic to true now…

We need to put mimic after verbose.